frontend/src/metabase/browse/components/ModelsTable.styled.tsx

```diff
@@ -8,11 +8,17 @@ import {
 import type { ResponsiveProps } from "metabase/components/ItemsTable/utils";
 import { breakpoints } from "metabase/ui/theme";
 
-export const ModelTableRow = styled.tr`
-  cursor: pointer;
+export const ModelTableRow = styled.tr<{ skeleton?: boolean }>`
   :outline {
     outline: 2px solid var(--mb-color-brand);
   }
+  ${props =>
+    props.skeleton
+      ? `
+    :hover { background-color: unset ! important; }
+    td { cursor: unset ! important; }
```

Comment:
i didn't think there could be a space between `!` and `important`. scared me fears some old browser might not like this so i might keep to the spec here.